src/main/java/org/jabref/logic/ai/AiDefaultPreferences.java

```diff
@@ -76,6 +77,45 @@ public class AiDefaultPreferences {
 
     public static final int CONTEXT_WINDOW_SIZE = 8196;
 
+    public static final Map<AiTemplate, String> TEMPLATES = Map.of(
+            AiTemplate.CHATTING_SYSTEM_MESSAGE,
+                    "You are an AI assistant that analyses research papers. You answer questions about papers.\n" +
+                            "You will be supplied with the necessary information. The supplied information will contain mentions of papers in form '@citationKey'.\n" +
+                            "Whenever you refer to a paper, use its citation key in the same form with @ symbol. Whenever you find relevant information, always use the citation key.\n\n" +
+                            "Here are the papers you are analyzing:\n" +
+                            "#foreach( $entry in $entries )\n" +
+                            "${CanonicalBibEntry.getCanonicalRepresentation($entry)}\n" +
+                            "#end",
+
+            AiTemplate.CHATTING_USER_MESSAGE, """
+                    $message
+
+                    Here is some relevant information for you:
+                    #foreach( $excerpt in $excerpts )
+                    ${excerpt.citationKey()}:
+                    ${excerpt.text()}
+                    #end""",
+
+            AiTemplate.SUMMARIZATION_CHUNK, """
+                Please provide an overview of the following text. It's a part of a scientific paper.
+                The summary should include the main objectives, methodologies used, key findings, and conclusions.
+                Mention any significant experiments, data, or discussions presented in the paper.
+
+                DOCUMENT:
+                $document
+
+                OVERVIEW:""",
```

Comment:
I don't understand that - this should be generated?

Reply:
About that. There are two ways of asking AI to generate text:

1. In form of chat: a list of messages. For this situation we need both system + user message templates.
2. In form of text that should be completed. This is only one template + I've seen that this approach is typically chosen for summarization. And I think it's easier.

For the summarization I chose 2nd approach. That is why you are seeing "OVERVIEW" and "FINAL OVERVIEW"

Reply:
Because without those words AI won't understand what should it do